wp-content/plugins/core/src/Admin/Editor/Classic_Editor_Formats.php

```diff
@@ -110,4 +110,25 @@ public function visual_editor_styles_dropdown( $settings ): array {
 		return $settings;
 	}
 
+	/**
+	 * Add minimal toolbar to use as default editor with custom blocks
+	 *
+	 * @filter acf/fields/wysiwyg/toolbars
+	 *
+	 * @param array $toolbars
+	 *
+	 * @return array
+	 */
+	public function add_minimal_toolbar( array $toolbars ): array {
+		$toolbars['Minimal'][1] = [
```

Comment:
See this is Uppercase, while the call to use it is lowercase. Is it possible to keep this consistent?

Also, since we have a ClassicEditor class, could we set the toolbar options as constants we can use so we can avoid typos?

Reply:
Yes and yes. By ClassicEditor class, you mean the current class, right? `Classic_Editor_Formats`?

Reply:
Yup!  We call it `$toolbars['Minimal']` here and `'toolbar'      => 'minimal',` later. If it was a class constant, it would be simple to reference and would avoid typos. like
```
$toolbars[Classic_Editor_Formats::minimal]
```

I just wasn't sure the the Uppercase and lowercase difference was a requirement.

Reply:
I understand now, thanks for the clarification. I will update the const and add to the blocks now.